### PR TITLE
feat: hello php nginx sample (multi-containers) 

### PR DESCRIPTION
**Pull request by pattishin**
**Issue:** To support documentation in https://cloud.google.com/run/docs/tutorials (php sample angle)

This uses new cloud build trigger `hello-php-nginx-sample-pr` and adds new e2e test.

### CODE REVIEW

**Comment by pattishin** [comment]
@JulienBreux When you have time to review, would love your feedback!

File: multi-container/hello-php-nginx-sample/php-app/index.php [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# [START cloudrun_mc_hello_php_nginx_sidecar_index]
+<?php
+  echo "This is main php app. Hello PHP World!";
+  phpinfo();
+?>
+# [END cloudrun_mc_hello_php_nginx_sidecar_index]
```
Comment (JulienBreux) [inline multi-container/hello-php-nginx-sample/php-app/index.php:6]:
```suggestion
<?php
// [START cloudrun_mc_hello_php_nginx_sidecar_index]

echo 'This is main php app. Hello PHP World!';

phpinfo();

// [END cloudrun_mc_hello_php_nginx_sidecar_index]
```

As above :)

**Review by JulienBreux — CHANGES_REQUESTED** [review]
Very good quality work!
We're delighted to be able to offer the PHP community a fine example of Cloud Run.
Big thanks!

**Comment by pattishin** [comment]
Since this is not lang agnostic, moving to `php-docs-samples`. 
https://github.com/GoogleCloudPlatform/php-docs-samples/pull/1901

Closing out here.

**Pull request closed without merging**